### PR TITLE
[ENH] Add normalisation transformer for time series

### LINKED ISSUE #605: [ENH] Normaliser/StandardScaler Transformer
### Describe the feature or idea you want to propose

we currently normalise series prior to clustering using scikit StandardScaler. However, this only works with 2D numpy and requires transposing series (since we want to normalise over a single series not over a single time point. This is messy to do with 3D numpy and easy to get wrong. Normalising along the wrong axis will completely skew results. I'm not a fan of wrapping scikit code, but in this instance I think it is worthwhile to do

### Describe your proposed solution

I think we need a CollectionTransformer that can normalise each series across each channel independently.  Perhaps just call it StandardScaler to avoid normalise/normalize arguments :)

### PR DESCRIPTION
**Pull request by TonyBagnall**
#### Reference Issues/PRs

Fixes #605 

#### What does this implement/fix? Explain your changes.

adds a wrapper for sktime.preprocessing.StandardScaler to make sure the transpose operations are handled correctly.

### CODE REVIEW

**Comment by aiwalter** [comment]
Can we actually not use the existing `sklearn` wrapper for this?

**Comment by TonyBagnall** [comment]
> Can we actually not use the existing `sklearn` wrapper for this?

its horrendously inefficent, see my comments on #605

**Review by hadifawaz1999 — APPROVED** [review]
LGTM!

File: aeon/transformations/collection/_scaler.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            or list of numpy arrays (any number of channels, unequal length series)
+                of shape [n_instances], 2D np.array (n_channels, n_timepoints_i), where
+                n_timepoints_i is length of series i
+
+        Returns
+        -------
+        Xt : same shape as input
+        """
+        X2 = [None] * len(X)
+        for i in range(len(X)):
```
[the remark this answers is not in the document]
Comment (hadifawaz1999) [reply aeon/transformations/collection/_scaler.py:101]: will keep this as a future enhance

**Pull request merged**